doc/reference/autoinstall-reference.rst

```diff
@@ -215,17 +201,13 @@ Identifier of the source to install (e.g., ``ubuntu-server-minimal``).
 .. _ai-network:
 
 network
--------
+~~~~~~~
 
 * **type:** Netplan-format mapping, see below
 * **default:** DHCP on interfaces named ``eth*`` or ``en*``
 * **can be interactive:** true
 
-`Netplan-formatted <https://netplan.readthedocs.io/en/stable/netplan-yaml/>`_ network configuration.
-This will be applied during installation as well as in the installed system.
-The default is to interpret the configuration for the installation media, which runs
-DHCP version 4 on any interface with a name matching ``eth*`` or ``en*`` but then
-disables any interface that does not receive an address.
+`Netplan-formatted <https://netplan.io/reference>`_ network configuration. This is applied during installation as well as in the installed system. The default is to interpret the configuration for the installation media, which runs DHCP version 4 on any interface with a name matching ``eth*`` or ``en*`` but then disables any interface that does not receive an address.
```

Comment:
(not blocking) The counter-argument here is that I'd like to use intersphinx to cross-reference to netplan, as it is one of several related yaml formats (with cloud-init and curtin, among others if we really build an extensive configuration) that can be part of a valid autoinstall.  In that case, where would we want to link in that document set?

Reply:
The redirects goes to https://netplan.readthedocs.io/en/latest/netplan-yaml/ now, which I believe is the right place -- the top of the `network` config reference. If ever they change the redirect (why?), we would need to reconsider. But until then, the redirect serves its ourpose I think, so we can use the "pretty" URL.